spec/models/grant_creator_request_spec.rb

```diff
@@ -1,5 +1,93 @@
 require 'rails_helper'
 
 RSpec.describe GrantCreatorRequest, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { is_expected.to respond_to (:requester) }
+  it { is_expected.to respond_to (:request_comment) }
+  it { is_expected.to respond_to (:reviewer) }
+  it { is_expected.to respond_to (:status) }
+
+  describe '#validations' do
+    let(:request)          { build(:grant_creator_request) }
+
+    it 'validates a valid request' do
+      expect(request).to be_valid
+    end
+
+    it 'requires a request_comment' do
+      request.request_comment = nil
+      expect(request).not_to be_valid
+      expect(request.errors).to include(:request_comment)
+    end
+
+    it 'disallows request from system_admin' do
+      request.requester.update_attribute(:system_admin, true)
+      expect(request).not_to be_valid
+      expect(request.errors[:base]).to include I18n.t('activerecord.errors.models.grant_creator_request.attributes.base.is_system_admin')
+    end
+
+    it 'disallows request from system_admin' do
```

Comment:
Should this label be someone who already has privilege.

Reply:
yes.